language-support/scala/bindings-akka/src/main/scala/com/digitalasset/ledger/client/binding/LedgerClientBinding.scala

```diff
@@ -56,7 +56,7 @@ class LedgerClientBinding(
 
     logger.debug(
       "[tx {}] subscription start with offset template selector {}, start {}, end {}",
-      party,
+      Party.unwrap(party),
```

Comment:
afaict, this is required as a side effect of https://github.com/scalaz/scalaz/pull/2076/files. @S11001001 Is there a better way of solving this?